packages/ember-htmlbars/tests/compat/make_bound_helper_test.js

```diff
@@ -101,7 +101,7 @@ QUnit.test('should update bound helpers in a subexpression when properties chang
   ignoreDeprecation(function() {
     view = EmberView.create({
       controller: { prop: 'isThing' },
-      template: compile('<div {{bind-attr data-foo=(dasherize prop)}}>{{prop}}</div>')
```

Comment:
This test is `should update bound helpers in a subexpression when properties change`, however here it has been changed to use an attribute expression and not a subexpression.

Can change this to still use a nested helper? It is specifically testing the subexpression hook.